Server/datalayer/time-entry.datalayer.js

```diff
@@ -1,12 +1,60 @@
+const timeentries = require('../models/time-entry.model');
 
 /*
  * @class timeEntriesDatalayer
  */
 class timeEntriesDatalayer {
 
     /**
-    * Initializes the datalayer
-    */
+     * Initializes the datalayer
+     */
+    static getTimeEntriesByUserId(obj) {
+
+        let param = {
+            "userId" : obj.userId,
+            "limit" : parseInt(obj.limit)
+        }
+        let sortOrder = {
+            '_id': -1
+        };
+
+        return timeentries
+            .find({"userId" : param.userId}, null, {limit: param.limit})
+            .sort(sortOrder)
+            .exec()
+            .catch((err) => {
+            if (err) throw new Error(`Could'nt fetch data with user ID ${userId}`);
```

Comment:
`If(err)` check is not required. Catch block signifies there is some error

Reply:
Updated